docs/logs/correlation/README.md

```diff
@@ -0,0 +1,25 @@
+# Correlate Logs with Traces
+
+Starting from `Microsoft.Extensions.Logging` version `5.0`, logs can be
+correlated with distributed tracing by enriching each log entry with the
+information from the enclosing `Activity`. This can be achieved by enabling the
+`ActivityTrackingOptions`:
+
+```csharp
+using var loggerFactory = LoggerFactory.Create(builder =>
```

Comment:
Most users of ILogger based logging are familiar with doing it from Asp.Net Core application. Those users will find it hard to figure out where do they make this changes in as Asp.Net Core application. Its not Otel's mistake, but we should consider console, aspnetcore separate for Ilogger docs.

Reply:
will discuss this separately to get this PR merged.